release_notes/Service_Fabric_ReleaseNotes_90CU2.md

```diff
@@ -0,0 +1,96 @@
+# Microsoft Azure Service Fabric 9.0 Cumulative Update 2.0 Release Notes
+
+This release includes bug fixes as described in this documents.
+
+The following packages and versions are part of this release:
+
+| Service | Platform | Version |
+|---------|----------|---------|
+|Service Fabric Runtime| Ubuntu 18 <br> Ubuntu 20 <br> Windows | 9.0.1056.1 <br> 9.0.1056.1 <br> 9.0.1048.9590  |
+|Service Fabric for Windows Server|Service Fabric Standalone Installer Package | 9.0.1048.9590 |
+|.NET SDK |Windows .NET SDK <br> Microsoft.ServiceFabric <br> Reliable Services and Reliable Actors <br> ASP.NET Core Service Fabric integration| 6.0.1048  <br> 9.0.1048 <br> 9.0.1048 <br> 9.0.1048 |
+|Java SDK  |Java for Linux SDK  | 1.0.6 |
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 11.0.1 |
+
+
+## Contents 
+
+Microsoft Azure Service Fabric 9.0 Cumulative Update 2.0 Release Notes
+
+* [Key Announcements](#key-announcements)
+* [Current Breaking Changes](#current-breaking-changes)
+* [Future Retirement and Deprecation Path Callouts](#future-retirement-and-deprecation-path-callouts)
+* [Service Fabric Common Bug Fixes](#service-fabric-common-bug-fixes)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+
+## Key Announcements
+* Windows Server 2022 is now supported as of the 9.0 CU2 release.
+* Mirantis Container runtime support on Windows for Service Fabric containers. To configure, see [Containers on Windows](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=dockerce#windows-server-1) documentation
+* The Microsoft Web Platform Installer (WebPI) used for installing Service Fabric SDK and Tools was retired on July 1, 2022. For latest guidance, see: [Set up Windows development environment](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started#to-use-visual-studio-2017-or-2019)
+
+## Current Breaking Changes
+
+* **Guest Executable and Container Applications**: Guest executable and container applications created or upgraded in Service Fabric clusters with runtime versions 7.1+ are incompatible with prior Service Fabric runtime versions (e.g. Service Fabric 7.0).<br/>
```

Comment:
Why is this being called out as it was announced in 8.2 CU3?

Reply:
Also, was there no content for the release notes from the 9.0 CU2 workitems?

Reply:
Added under Service Fabric Common Bug Fixes: